### ISSUE TITLE
What about YOLOv8?

### ISSUE THREAD

**Issue by Egorundel** [ultralytics/ultralytics#17821]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Do you only support YOLOv11 now? How do I train a neural network if I want YOLOv8?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @Egorundel, thank you for your interest in Ultralytics 🚀! We recommend checking out the [Docs](https://docs.ultralytics.com), where you can find comprehensive resources, including [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples, covering YOLOv8 and other versions.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) so we can better assist with debugging.

If this is a❓Question, it’s worth noting that the Ultralytics repository supports multi-version YOLO workflows, including YOLOv8 and YOLOv11. Please share more details about the challenges or goals you’re working on to help us better understand your needs.

Dive into the Ultralytics community where you feel most at home: connect on [Discord](https://ultralytics.com/discord) 🎧 for real-time discussions, join deeper dialogues on [Discourse](https://community.ultralytics.com), or visit our [Subreddit](https://reddit.com/r/ultralytics) to exchange knowledge with fellow developers.  

## Upgrade

It’s always a great idea to upgrade to the latest version before proceeding. Update the `ultralytics` package with the following command to ensure the newest features and fixes are in place:

```bash
pip install -U ultralytics
```

Ensure your environment is set up with [**Python>=3.8**](https://www.python.org/) and [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to avoid compatibility issues. For additional details, refer to the [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml).

## Environments

YOLO is versatile and can be run on various verified environments:

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>  

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) tests are passing. These tests validate the functionality of YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) across macOS, Windows, and Ubuntu.

⚠️ Please note this is an automated response to help direct you to key resources. An Ultralytics engineer will review your inquiry and provide further assistance soon. Thank you for your patience and contribution! 😊

**Comment by Y-T-G** [comment]
`ultralytics` supports multiple models including YOLOv8. You can find all of them in the docs.

https://docs.ultralytics.com/models

**Comment by Y-T-G** [comment]
Duplicate of https://github.com/ultralytics/ultralytics/issues/16731

> ### Search before asking
> 
> - [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.
